### ISSUE TITLE
Some crash logs

### ISSUE THREAD

**Issue by Winterwolf** [romovs/amber#60]
For last 2 days my client crashed 3 times. There are logs:

```
1.60.0.e5f9bf0b
Linux 4.10.0-28-generic, 1.8.0_131 amd64
NVIDIA Corporation (GeForce GTX 465/PCIe/SSE2) - 4.5.0 NVIDIA 381.22

haven.UI$UIException: Null parent widget 5 for 6
	at haven.UI.newwidget(UI.java:157)
	at haven.RemoteUI.run(RemoteUI.java:66)
	at haven.MainFrame.run(MainFrame.java:268)
	at java.lang.Thread.run(Thread.java:748)

haven.UI$UIException: Null parent widget 5 for 8
	at haven.UI.newwidget(UI.java:157)
	at haven.RemoteUI.run(RemoteUI.java:66)
	at haven.MainFrame.run(MainFrame.java:268)
	at java.lang.Thread.run(Thread.java:748)

java.lang.RuntimeException: Unknown position operation: h
	at haven.Widget.relpos(Widget.java:402)
	at haven.Widget.addchild(Widget.java:412)
	at haven.Widget.makechild(Widget.java:420)
	at haven.UI.newwidget(UI.java:159)
	at haven.RemoteUI.run(RemoteUI.java:66)
	at haven.MainFrame.run(MainFrame.java:268)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by romovs** [comment]
This should be fixed in the new release 1.61.0

**Comment by Winterwolf** [comment]
Thank you!
